Update getting-started-part-three.md (#1562)
codeship/documentation: _pro/quickstart/getting-started-part-three.md

### Required Services 

#### Project linked to a Git Repository

> Within your SCM of choice you need to have configured the directory where you have been making these files as a Repository. 

Reviewer: Would lowercase repository

> Within your SCM of choice you need to have configured the directory where you have been making these files as a Repository. 

Reviewer 2: ```suggestion
Within your SCM of choice you need to have configured the directory where you have been making these files as a repository. 
```

> You also need to have linked this Repository in your SCM to a CodeShip Pro Project at [CodeShip UI](https://app.codeship.com/home)

Reviewer: Lowercase repository and project - would also link to the project page instead (https://app.codeship.com/projects)

> You also need to have linked this Repository in your SCM to a CodeShip Pro Project at [CodeShip UI](https://app.codeship.com/home)

Reviewer 2: ```suggestion
You also need to have linked this repository in your SCM to a CodeShip Pro project at [CodeShip UI](https://app.codeship.com/home)
```

The newer versions of Docker have changed to store credentials in the MacOS keychain rather than in a configuration file.

To get the appropriate authentication file on MacOS, you will need to remove the `credsStore` line from Docker's `config.json` to disable Keychain storage, re-run `docker login` and then use the values it then generates in your updated `dockercfg` as shown above.

> **NOTE:** If this method to retrieve your authentication does not work you may manually generate your authentication key by running the following command in your terminal: `echo -n 'DOCKER_HUB_USERNAME:DOCKER_HUB_PASSWORD' | base64` and then copy this encrypted auth key to your `dockercfg` file. 

Reviewer 2: ```suggestion
**NOTE:** If this method to retrieve your authentication does not work, you may manually generate your authentication key by running the following command in your terminal: `echo -n 'DOCKER_HUB_USERNAME:DOCKER_HUB_PASSWORD' | base64` and then copy this encrypted authentication key to your `${HOME}/.docker/config.json` file. 
```
